modules/extensions/partials/assembly_eclipse-che4z.adoc

```diff
@@ -125,10 +125,10 @@ https://join.slack.com/t/che4z/shared_invite/enQtNzk0MzA4NDMzOTIwLWIzMjEwMjJlOGM
 
 Zowe Command Line-Interface (Zowe CLI) is a command-line interface that lets application developers interact with the mainframe in a familiar format. 
 // The following Zowe CLI plug-ins are included in the Che4z basic stack:
-// * https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-mainframe-software/devops/ca-brightside/3-0/ca-brightside-command-line-interface-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ca-endevor-scm.html[CA Endevor SCM plug-in for Zowe CLI]
-// * https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-mainframe-software/devops/ca-brightside/3-0/ca-brightside-command-line-interface-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ca-file-master-plus.html[CA File Master Plus plug-in for Zowe CLI]
-// * https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-mainframe-software/devops/ca-brightside/3-0/ca-brightside-command-line-interface-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-db2-database.html[IBM Db2 plug-in for Zowe CLI]
-// * https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-mainframe-software/devops/ca-brightside/3-0/ca-brightside-command-line-interface-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-cics.html[IBM CICS plug-in for Zowe CLI]
+// * https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-endevor-scm-plug-in-for-zowe-cli.html[CA Endevor SCM plug-in for Zowe CLI]
+// * https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-file-master-plus-plug-in-for-zowe-cli.html[CA File Master Plus plug-in for Zowe CLI]
+// * https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-db2-database.html[IBM Db2 plug-in for Zowe CLI]
+// * https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-cics.html[IBM CICS plug-in for Zowe CLI]
```

Comment:
The links are commented out and don't appear in the docs. If they need to appear again, uncomment them, like this:

```suggestion
 * https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-endevor-scm-plug-in-for-zowe-cli.html[CA Endevor SCM plug-in for Zowe CLI]
 * https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-file-master-plus-plug-in-for-zowe-cli.html[CA File Master Plus plug-in for Zowe CLI]
 * https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-db2-database.html[IBM Db2 plug-in for Zowe CLI]
 * https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-cics.html[IBM CICS plug-in for Zowe CLI]
```